README.md

```diff
@@ -100,40 +100,36 @@ https://arewecompressedyet.com/analyzer/?d=https://people.xiph.org/~mbebenita/an
 
 ## Coding style
 Check code formatting with [rustfmt](https://github.com/rust-lang-nursery/rustfmt) before submitting a PR.
-rav1e currently uses the nightly version of rustfmt.
+rav1e currently uses the stable version of rustfmt.
 
-To install nightly:
+To install stable tool chain:
 
 ```
-rustup install nightly
+rustup install stable
 ```
 
-To install the nightly version of rustfmt:
+To install the rustfmt:
 
 ```
-rustup component add rustfmt-preview --toolchain nightly
+rustup component add rustfmt
 ```
 
 then
 
 ```
-cargo +nightly fmt -- --check
+cargo fmt -- --check
 ```
 
 You should also try [clippy](https://github.com/rust-lang-nursery/rust-clippy).
```

Comment:
"you should also try" sounds weird. It's quite important, so perhaps it could be a recommendation?